Assignment2-Python/stackmin.py

```diff
@@ -0,0 +1,39 @@
+#3.2 Stack Min (p.98)
+'''Stack Min: How would you design a stack which, in addition to p u s h and pop, has a function m i n
+which returns the minimum eiement? Push, p o p and m i n should ail operate in 0 ( 1 ) time'''
```

Comment:
The general spacing for multiline comments is to put the quotation marks on their own line.
```suggestion
"""
3.2 Stack Min (p.98)
Stack Min: How would you design a stack which, in addition to push and pop, has a function min
which returns the minimum eiement? Push, pop and min should ail operate in O(1) time
"""
```